src/main/java/com/codeofus/rent_a_park/controllers/PersonController.java

```diff
@@ -21,27 +22,37 @@
 public class PersonController {
 
     PersonService personService;
-    ParkingMapper mapper;
+    PersonMapper personMapper;
+
+    @GetMapping
+    public List<PersonDto> getAllPersons(Pageable pageable) {
+        List<PersonInfo> personInfos = personService.getAllPersons(pageable);
+        return personMapper.personInfoListToPersonDtoList(personInfos);
+    }
+
+    @GetMapping("/{id}")
+    public PersonDto getPerson(@PathVariable int id) throws BadRequestException {
+        Optional<Person> person = personService.getPerson(id);
+        if (person.isPresent()) {
+            return personMapper.personToPersonDTO(person.get());
+        } else {
+            throw new BadRequestException("Person does not exist", "persons", "does-not-exist");
```

Comment:
better to handle this at service layer (although the name of the exception isn't great then)